src/touch_light.ino

```diff
@@ -111,7 +113,7 @@ int rPin = D3;
 unsigned char myId = 0;
 
 int currentEvent = tEVENT_NONE;
-int eventTime = Time.now();
+int eventTime = 0;
```

Comment:
Note that seeding this with Time.now() means that it is impossible to grab the current color it should be and it just stays dark until a light is touched somewhere.